.emacs.d/lisp/code/languages/language-hub.el

```diff
@@ -1,7 +1,8 @@
 (defvar languages
   '(c
     elisp
-    ruby)
+    ruby
+    web-c)
```

Comment:
`web` is a package.  Since this is a configuration for `web-mode`, I figured `web-c` was appropriate.